SU-SWS/saa_victoria_subtheme
# ADAPT-482 Update lockup line height for specific breakpoints

# READY FOR REVIEW

# Summary
- Update lockup line height for specific breakpoints

# Review By (Date)
- today

# Urgency
- high

# Steps to Test

1. Pull this branch
2. Clear site cache
3. View 992 screen width. The line height should be 1.2em.

# Associated Issues and/or People
- [ADAPT-482](https://stanfordits.atlassian.net/browse/ADAPT-482)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
